.github/workflows/checks.yml

```diff
@@ -0,0 +1,39 @@
+name: "PR Checks"
+
+on: [pull_request]
```

Comment:
we should also run these on push to master

Reply:
https://github.com/ExodusMovement/fork-sync/pull/4/commits/d0a7657eaae7f8f6d604acaa2d8400a110c98aac